Core/Inc/datastructs.h

```diff
@@ -12,7 +12,7 @@
  */
 typedef struct {
 	/* These are retrieved from the initial LTC comms */
-	uint16_t voltage_reading[NUM_CELLS_PER_CHIP];	/* store voltage readings from each chip */
+	uint16_t voltage[NUM_CELLS_PER_CHIP];	/* store voltage readings from each chip */
```

Comment:
Why the rename? The `reading` vs `value` convention allows us to store both raw readings and filtered values